src/nanodbc.h

```diff
@@ -1461,6 +1461,26 @@ class catalog
         result result_;
     };
 
+     //! \brief Result set for a list of tables and the privileges associated with each table.
+    class table_privileges
+    {
+    public:
+        bool next();
```

Comment:
If we add `///<` style comments here it will allow Doxygen to pickup these fields in the generated documentation files.